commerce.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
from my experience i would highly recommend to not put this here as a main package, as it will throw errors as there is no `main` method. `go get flamingo.me/flamingo-commerce/v3` would fail because it would try to compile a non-existing binary

Reply:
Moved to docs/openapidoc.go

Reply:
Moving to subfolder did not work (generation failed)... I added it as main_test..

Other suggestions @bastianccm